repo/packages/H/hdfs/0/marathon.json

```diff
@@ -4,12 +4,12 @@
   "mem": 512,
   "disk": 100,
   "instances": 1,
-  "cmd": "{{#hdfs.custom-config}}echo {{hdfs.custom-config}} | base64 --decode > mesos-site.xml && {{/hdfs.custom-config}}cd hdfs-mesos* && if [ -e '../mesos-site.xml' ]; then cp -f ../mesos-site.xml ./etc/hadoop/; fi && ./bin/hdfs-mesos",
+  "cmd": "export LD_LIBRARY_PATH=/opt/mesosphere/lib && export JAVA_HOME=$MESOS_DIRECTORY/jre1.7.0_76 && env && {{#hdfs.custom-config}}echo {{hdfs.custom-config}} | base64 --decode > mesos-site.xml && {{/hdfs.custom-config}}cd hdfs-mesos-* && if [ -e '../mesos-site.xml' ]; then cp -f ../mesos-site.xml ./etc/hadoop/; fi && ./bin/hdfs-mesos",
   "uris": {{{hdfs.scheduler-uris}}},
```

Comment:
Also, instead of export, we should definitely use env variables. See https://mesosphere.github.io/marathon/docs/rest-api.html to see how a user was using LD_LIBRARY_PATH as an env variable with their cmd